### PR TITLE
[71] added downdip coarsening

### PR DESCRIPTION
**Pull request by voj**
implements #71

### CODE REVIEW

File: src/main/java/nz/cri/gns/NSHM/opensha/util/NSHMPythonGateway.java [hunk 1]
```diff
@@ -51,15 +52,37 @@ public NSHMRuptureSetBuilder setPermutationStrategy(String permutationStrategyCl
...
+			return this;
+		}
 		
 		/**
 		 * Caches the results of the build
 		 */
 		@Override
-		public SlipAlongRuptureModelRupSet buildRuptureSet(String fsdFileName) throws DocumentException, IOException {
-			ruptureSet = super.buildRuptureSet(fsdFileName);
+		public SlipAlongRuptureModelRupSet buildRuptureSet() throws DocumentException, IOException {
```
Comment (chrisbc) [inline src/main/java/nz/cri/gns/NSHM/opensha/util/NSHMPythonGateway.java:81]: @voj looks like the Gateway python clients will need a tweak for this change. I'll test for this and commit changes required before I approve - cheers

**Review by chrisbc — COMMENTED** [review]
looks good Oakley thanks. I'm just going to check the python API change before I approve.

File: src/main/java/nz/cri/gns/NSHM/opensha/util/NSHMPythonGateway.java (hunk 1 above)
Comment (voj) [reply src/main/java/nz/cri/gns/NSHM/opensha/util/NSHMPythonGateway.java:81]: 👍

Comment (chrisbc) [reply src/main/java/nz/cri/gns/NSHM/opensha/util/NSHMPythonGateway.java:81]: No changes actually - my bad. All working as usual :+1:

**Pull request merged**